src/backend/utils.ts

```diff
@@ -919,11 +920,12 @@ export async function downloadDefaultWine() {
   // get list of wines on wineDownloaderInfoStore
   const availableWine = wineDownloaderInfoStore.get('wine-releases', [])
   // use GE-Proton type if on Linux and GamePortingToolkit if on Mac
-  const release = availableWine.find((version) => {
+  const release = availableWine.find(async (version) => {
     if (isLinux) {
       return version.type === 'GE-Proton'
     } else if (isMac) {
-      if (isIntelMac) {
+      const isMacOSUpToDate = await isMacSonomaOrHigher()
+      if (isIntelMac && isMacOSUpToDate) {
```

Comment:
is this check correct? because this means that we'd set wine-crossover only if intel mac and new OS version, when I think we want to only set game porting toolkit if NOT intel mac and up-to-date os

we actually want wine-crossover always for intel macs

Reply:
TRue, should be !isMacOsUpToDate then. good catch.